project-codeflare/codeflare-sdk
# mv demo nbs into src folder and add fn for cloning to current dir

# Issue link
https://issues.redhat.com/browse/RHOAIENG-10266

# What changes have been made
Be not afraid of the many files being changed, they are all just being moved from the root to be inside the sdk src dir

# Verification steps

1. build package via `poetry build`
2. create venv and activate it
3. install built wheel by doing `pip install `<path-to-sdk-dir>/dist/codeflare_sdk-0.0.0.dev0-py3-none-any.whl`
4. activate python shell
5. run `import codeflare_sdk`
6. run `codeflare_sdk.clone_demo_nbs()`
7. verify that `demo-notebooks` dir is created with all demo nbs present

## Checks
- Testing Strategy
